content/automate/ManageIQ/Infrastructure/VM/Migrate/StateMachines/Methods.class/__methods__/checkmigration.rb

```diff
@@ -36,9 +39,12 @@ def checkmigration(task)
                   reason = @handle.root['vm_migrate_task'].message
                   reason = reason[7..-1] if reason[0..6] == 'Error: '
                   @handle.root['ae_reason'] = reason
+                when 'failure'
+                  @handle.root['ae_result'] = 'error'
+                  @handle.root['ae_reason'] = "Migration failed"
                 when 'retry'
```

Comment:
you need retry when result is 'active'

Reply:
This would mean that we need to retry from [here](https://github.com/ManageIQ/manageiq-content/pull/197/files#diff-04a70eac4f0eb3c42634d50eafda6dfcR49) which seems not the best idea.

Reply:
or from the line you commented